Way Too Long Words/Way Too Long Words/Program.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System.Text;
+
+namespace Way_Too_Long_Words
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Reader reader = new Reader();
+
+            var numOfWord = reader.NextInt();
+            StringBuilder result = new StringBuilder();
+            while (numOfWord != 0)
+            {
+                var word = reader.Next();
+                if (word.Length > 10)
+                {
+                    result.Append(AbbWord(word));
+                }
+                else
+                {
+                    result.Append(word);
+                }
+
+                result.Append("\n");
+                numOfWord--;
+            }
+            Console.Write(result.ToString());
+        }
+        static StringBuilder AbbWord(string word)
```

Comment:
We should add an access modifier for this method. Besides, the method's name is not clear for me.